yarn-project/archiver/src/archiver/archiver_store_test_suite.ts

```diff
@@ -374,11 +374,11 @@ export function describeArchiverDataStore(testName: string, getStore: () => Arch
         new Fr((blockNumber * 100 + txIndex * 10 + logIndex) * (isPublic ? 123 : 1));
 
       // See parseLogFromPublic
-      const makeLengthsField = (publicValuesLen: number, privateValuesLen: number, ciphertextLen: number) => {
+      // Search the codebase for "disgusting encoding" to see other hardcoded instances of this encoding, that you might need to change if you ever find yourself here.
```

Comment:
As much as I dislike leaving some of these hard-coded constants here (that I inherited), I'm going to leave them, because this will imminently be obliterated by partial note log refactoring.